pkg/download/download.go

```diff
@@ -70,6 +71,9 @@ func Download(uri, destination string, mode os.FileMode, sha256sum []byte) (stri
 	if err != nil {
 		return "", errors.Wrapf(err, "unable to get request from %s", uri)
 	}
+	if ctx != nil {
```

Comment:
Can this be `nil`?

Reply:
See reply above

Reply:
Changed to a panic call when context is `nil`